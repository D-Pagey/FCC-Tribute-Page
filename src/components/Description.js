import React, { Component } from 'react';

class Description extends Component {
  render() {
    return (
      <div>
        <hr className="horizontal-rule hr-margin" />
        <p className="text copy">Chris Sharma is an American rock climber. In 2007, NPR wrote that Sharma
          had been described as the worlds best rock climber. He is known as the master
          of climbing; by making first ascents of many of the hardest lines of the
          Catalu√±a region in Spain and by opening Sharma Climbing BCN in Barcelona in 2015.</p>
        <hr className="horizontal-rule hr-margin" />
      </div>
    )
  }
}

export default Description;
