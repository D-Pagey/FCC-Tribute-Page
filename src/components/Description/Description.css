.horizontal-rule {
  align-self: center;
  border: 0;
  border-top: 1px solid;
  color: rgba(0, 0, 0, 0.1);
  margin: .01em 0 1em;
  width: 92.5%; }

.hr-margin {
  margin: 1.25em 0;
  width: 100%; }

  .description {
    align-self: center;
    width: 92.5%;
  }
