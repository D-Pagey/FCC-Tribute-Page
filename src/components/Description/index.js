import React from 'react';

import './index.css';

export default function Description() {
  return (
    <div className="description">
      <hr className="horizontal-rule" />
      <p>Chris Sharma is an American rock climber. In 2007, NPR wrote that Sharma
        had been described as the worlds best rock climber. He is known as the master
        of climbing; by making first ascents of many of the hardest lines of the
        Catalu√±a region in Spain and by opening Sharma Climbing BCN in Barcelona
        in 2015.</p>
      <hr className="horizontal-rule" />
    </div>
  )
}
