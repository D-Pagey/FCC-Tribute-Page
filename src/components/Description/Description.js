import React, { Component } from 'react';

import './Description.css';

class Description extends Component {
  render() {
    return (
      <div className="description">
        <hr className="horizontal-rule hr-margin" />
        <p>Chris Sharma is an American rock climber. In 2007, NPR wrote that Sharma
          had been described as the worlds best rock climber. He is known as the master
          of climbing; by making first ascents of many of the hardest lines of the
          Catalu√±a region in Spain and by opening Sharma Climbing BCN in Barcelona in 2015.</p>
        <hr className="horizontal-rule hr-margin" />
      </div>
    )
  }
}

export default Description;
