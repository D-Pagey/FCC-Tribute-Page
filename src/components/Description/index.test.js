import React from 'react';
import { shallow } from 'enzyme';

import Description from '.';

test('render a Description', () => {
  const wrapper = shallow(
    <Description />,
  );
  expect(wrapper).toMatchSnapshot();
});
