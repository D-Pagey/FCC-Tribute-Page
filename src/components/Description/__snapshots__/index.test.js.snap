// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render a Description 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Description />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <hr
          className="horizontal-rule"
/>,
        <p>
          Chris Sharma is an American rock climber. In 2007, NPR wrote that Sharma had been described as the worlds best rock climber. He is known as the master of climbing; by making first ascents of many of the hardest lines of the Cataluña region in Spain and by opening Sharma Climbing BCN in Barcelona in 2015.
</p>,
        <hr
          className="horizontal-rule"
/>,
      ],
      "className": "description",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "className": "horizontal-rule",
        },
        "ref": null,
        "rendered": null,
        "type": "hr",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Chris Sharma is an American rock climber. In 2007, NPR wrote that Sharma had been described as the worlds best rock climber. He is known as the master of climbing; by making first ascents of many of the hardest lines of the Cataluña region in Spain and by opening Sharma Climbing BCN in Barcelona in 2015.",
        },
        "ref": null,
        "rendered": "Chris Sharma is an American rock climber. In 2007, NPR wrote that Sharma had been described as the worlds best rock climber. He is known as the master of climbing; by making first ascents of many of the hardest lines of the Cataluña region in Spain and by opening Sharma Climbing BCN in Barcelona in 2015.",
        "type": "p",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "className": "horizontal-rule",
        },
        "ref": null,
        "rendered": null,
        "type": "hr",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <hr
            className="horizontal-rule"
/>,
          <p>
            Chris Sharma is an American rock climber. In 2007, NPR wrote that Sharma had been described as the worlds best rock climber. He is known as the master of climbing; by making first ascents of many of the hardest lines of the Cataluña region in Spain and by opening Sharma Climbing BCN in Barcelona in 2015.
</p>,
          <hr
            className="horizontal-rule"
/>,
        ],
        "className": "description",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "className": "horizontal-rule",
          },
          "ref": null,
          "rendered": null,
          "type": "hr",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Chris Sharma is an American rock climber. In 2007, NPR wrote that Sharma had been described as the worlds best rock climber. He is known as the master of climbing; by making first ascents of many of the hardest lines of the Cataluña region in Spain and by opening Sharma Climbing BCN in Barcelona in 2015.",
          },
          "ref": null,
          "rendered": "Chris Sharma is an American rock climber. In 2007, NPR wrote that Sharma had been described as the worlds best rock climber. He is known as the master of climbing; by making first ascents of many of the hardest lines of the Cataluña region in Spain and by opening Sharma Climbing BCN in Barcelona in 2015.",
          "type": "p",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "className": "horizontal-rule",
          },
          "ref": null,
          "rendered": null,
          "type": "hr",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
